src/views/v2/month/day.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * View: Month View - Day
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+// @todo: This is just for presentation purposes, while building the view.
+$day_number  = ( $week < 1 ) ? ( $day + 1 ) * ( $week + 1 ) : ( $day + 1 ) + $week * 7;
+
+$day_title_classes = [ 'tribe-events-calendar-month__day-date' ];
+$day_id = 'tribe-events-calendar-day-' . $day_number;
```

Comment:
We should maybe add the month number to this too, so it's like `'tribe-events-calendar-day-' . $month_number . '-' . $day_number` (or any other way to create a unique id). This will result in gridcells with the same id at edge cases like the start and end of the month.

Reply:
Yes, the thing is that we don't have the month number, and it's just data with testing purposes. We want to hardcode this stuff just for the sake of doing it?